scripts/restore

```diff
@@ -27,14 +27,11 @@ ynh_script_progression --message="Restoring the data directory..." --weight=1
 ynh_restore_file --origin_path="$data_dir" --not_mandatory
 
 #=================================================
-# (RE)INSTALL NPM MODULE
+# INSTALL DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing Peertube remote runner npm module..." --weight=1
+ynh_script_progression --message="Installing dependencies..."
 
-pushd "$install_dir"
-  ynh_use_nodejs
-  ynh_exec_warn_less ynh_exec_as $app env $ynh_node_load_PATH NODE_CONFIG_DIR="$install_dir/config" NODE_ENV=production $ynh_npm install @peertube/peertube-runner
-popd
+ynh_exec_warn_less ynh_install_nodejs --nodejs_version=$nodejs_version
```

Comment:
Why ?
Does that change the functionality ?

Reply:
Restore script is not meant to be install script. we are restoring `install_dir` path with peertube-runner already installed in the install script

Reply:
I saw the rest of you changes afterwards, now I understand them :)

However in that case restoring or install is basically the same (except install might have a never version), as there is no particular thing to backup, so I wonder if the restore script is even necessary :thinking: